x/xgit/repository_test.go

```diff
@@ -8,6 +8,22 @@ import (
 	"gopkg.in/src-d/go-git.v4/plumbing"
 )
 
+func TestIsGitURL(t *testing.T) {
```

Comment:
if we can add some test cases for non-git urls that would be nice

Reply:
But it would require access to the network and that won't be good.